pinc/Pool.inc

```diff
@@ -35,17 +35,18 @@ class Pool extends Stage
      *   An array of strings to echo on the pool's home page.
      */
     public function __construct(
-        $id,
-        $name,
-        $access_minima,
-        $after_satisfying_minima,
-        $evaluation_criteria,
-        $access_change_callback,
-        $description,
-        $document,
-        $foo_Header,
-        $foo_field_name,
-        $blather
+        string $id,
```

Comment:
I added this type to prove that that the `$id` parameter is a string and thus we need to manually ignore some issue in PHPStan on `pinc/stages.inc`.

The other types in the constructors were added bonuses.

All the types are copied from the class member's.